### PR TITLE
Feature/id_57/Event Management

### PR DESCRIPTION
**Pull request by gabitrujillo**

### CODE REVIEW

File: apps/cevent_frontend/src/app/my_events/[id]/_components/Sections/HomeSection.tsx [hunk 1]
```diff
@@ -1,31 +1,191 @@
...
+                alignItems: 'center',
+              }}
+            >
+              <Typography variant="h6">Event Status</Typography>
+              <Select
+                value={eventStatus}
+                onChange={handleStatusChange}
+                sx={{ width: '150px' }}
+              >
+                <MenuItem value="Pending">Pending</MenuItem>
```
Comment (CatCoder36) [inline apps/cevent_frontend/src/app/my_events/[id]/_components/Sections/HomeSection.tsx:174]: Please add all available event states

File: apps/cevent_frontend/src/app/my_events/[id]/_components/Sections/HomeSection.tsx [hunk 2]
```diff
@@ -1,31 +1,191 @@
...
+              <Grid size={6}>
+                <Card sx={stylesPage.ticketCard}>
+                  <Typography>Tickets Sold</Typography>
+                  <Typography>{event.attendeeCount}</Typography>
+                </Card>
+              </Grid>
+              <Grid size={6}>
+                <Card sx={stylesPage.ticketCard}>
+                  <Typography>Tickets Redeemed</Typography>
+                  <Typography>{event.attendeeCount}</Typography>
```
Comment (CatCoder36) [inline apps/cevent_frontend/src/app/my_events/[id]/_components/Sections/HomeSection.tsx:196]: Tickets redeemed are not equal to tickets sold, currently this counter depends on other user stories, try to leave a TODO type comment, example:

{///TODO: Replace when ticket sale is implemented}

**Comment by CatCoder36** [comment]
Update to develop

**Comment by JP-Arequipa** [comment]
image

The location URL is not working, it should redirect to google maps

**Comment by JP-Arequipa** [comment]
image

The status is not been updated correctly, check if it is a UI refresh bug or backend update fail

**Comment by JP-Arequipa** [comment]
https://github.com/user-attachments/assets/e001ef7c-8790-423e-a084-74c345390717

The event is not been updated well in the backend, when you change de status and refresh the UI it is not showing the updated status

**Review by JP-Arequipa — APPROVED** [review]

**Pull request merged**